x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EsqlActionIT.java

```diff
@@ -562,11 +555,6 @@ public void testFilterWithNullAndEvalFromIndex() {
         assertThat(results.columns(), hasItem(equalTo(new ColumnInfo("data", "long"))));
         assertThat(results.columns(), hasItem(equalTo(new ColumnInfo("data_d", "double"))));
         assertThat(results.columns(), hasItem(equalTo(new ColumnInfo("time", "long"))));
-
-        // restore index to original pre-test state
```

Comment:
No need to delete docs in these tests. The index should be deleted after each test.